packages/core/chrome/core-chrome-browser-internal/src/ui/header/collapsible_nav.scss

```
@@ -1,9 +1,29 @@
+$screenHeightBreakpoint: 15em;
```

After looking at the rest of the file, I think we may still want to use a variable for this one rather than a static number, as a general best practice. Can you try, e.g.

```suggestion
$screenHeightBreakpoint: $euiSize * 15;
```
And see if that keeps the collapse behavior the same as before?

---

I just tested this locally and this gives us the same behavior that allows us to traverse the menu.